apps/admin-server/src/pages/projects/[project]/settings/map.tsx

```diff
@@ -98,6 +104,14 @@ export default function ProjectSettingsMap() {
     }
   }, [ form.watch('minZoom'), form.watch('maxZoom') ] );
 
+  const tileLayerOptions = [
+    { value: 'nlmaps', label: 'Nederlandse Kaart' },
+    { value: 'amaps', label: 'Amsterdam Kaart' },
+    { value: 'openstreetmaps', label: 'OpenStreetMap' },
+    { value: 'n3s', label: 'CartoDB Light' },
```

Comment:
Zitten deze standaard in de Leaflet? Of mis ik hier nog code om dit op te vangen in de kaart?